README.md

```diff
@@ -102,3 +102,51 @@ const InstanceLogger = init(({ name }) => {
 
 const george = InstanceLogger({ name: 'George' }); // Created new instance: George
 ```
+
+
+### overrides()
```

Comment:
Perhaps add here a link to [article](https://medium.com/@koresar/fun-with-stamps-episode-3-comparing-with-the-es2015-classes-e387ef041896) to show a real use case.